hardware/is_camera/src/camera.cc

```diff
@@ -275,26 +274,58 @@ namespace is_camera {
       }
     }
 
-    if (!camera.GetBool("bayer_enable", &bayer_enable_)) {
+    bool bayer_enable;
+    if (!camera.GetBool("bayer_enable", &bayer_enable)) {
       FF_FATAL("Bayer enable not specified.");
       exit(EXIT_FAILURE);
     }
+    // Check if bayer configuration changed
+    EnableBayer(bayer_enable);
+
+    // Auto Exposure Parameters
+    bool auto_exposure;
+    if (!camera.GetBool("auto_exposure", &auto_exposure)) {
+      FF_FATAL("Auto Exposure enable not specified.");
+      exit(EXIT_FAILURE);
+    }
+    if (auto_exposure && !auto_exposure_) auto_exposure_timer_.start();
```

Comment:
race condition (although unlikely) - should start timer after below parameters initialized